packages/testlab/src/client.ts

```diff
@@ -26,22 +26,29 @@ export function createClientForHandler(
   return supertest(server);
 }
 
+export interface RestServer {
+  options?: {
+    http?: {
+      port?: number;
+    },
+  };
+  start() : Promise<void>;
+}
+
 export interface Server {
-  config: {port: number};
-  start(): Promise<void>;
+  handleHttp(req: http.ServerRequest, res: http.ServerResponse): void;
```

Comment:
This is REST-specific, I think it should go to `RestServer`?

Reply:
Originally, it was, but Lerna sees this and creates circular links during bootstrapping. I had to make up a compatible-but-separate interface here to avoid this; the circular linking was actually causing real issues during test runs!

Reply:
Nvm, I think I misunderstood. I have the two interfaces mixed up, I'll take a crack at sorting it out.